package.json

```diff
@@ -57,21 +58,21 @@
   "author": "Amazon Web Services",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@babel/core": "^7.1.6",
-    "@babel/preset-env": "^7.1.6",
-    "babel-cli": "^6.26.0",
```

Comment:
Is this intended? for 'removing line 62'

Reply:
Yes. This dependency is not used and it conflicts with the upgrade of another dependency so I removed it.